### PR TITLE
Finished up all of my builders

### PR DESCRIPTION
**Pull request by NickAnson**

### CODE REVIEW

File: edu/jalc/automobile/onlinebuilder/components/suspension/economy/EconomySuspension.java [hunk 1]
```diff
@@ -49,6 +49,7 @@ public String toString(){
       return "EconomySuspension has :\n" +
               "\t"+ stockShock.toString()+"\n"+
               "\t"+ stockSpring.toString()+"\n"+
+              "\t"+ wheel.toString() +"\n" +
```
Comment (NickAnson) [inline edu/jalc/automobile/onlinebuilder/components/suspension/economy/EconomySuspension.java:52]: Threw my code off for hours due to this not being implemented in this already haha.

**Comment by ClarkLindsay** [comment]
A few small notes. First, when constructing the engine, the first double value you input is supposed to represent the liter size. So a 2.0-liter engine will have 2.0 there; however, a 1.4-liter engine will have 1.4.
Also, I reread the ReadMe for this lab and saw a huge error that I had overlooked. The builders need to implement a DodgeDartBuilder, not a DodgeRamBuilder. This is a huge error and would have cost us 20 points; however, it is only a two line change. Just pull my repository and implement the DodgeDartBuilder. Last, I think you needed three builders standard and one more for extra credit. I only see two. Thanks.

-Clark

**Comment by ClarkLindsay** [comment]
Never mind about the other builders, I found them. One note, do you have UMLs somewhere for all of this? You can check what I did (it is super simple). Thanks

**Comment by NickAnson** [comment]
Okay i changed what i think you were talking about in my code and my umls are in the issues

**Comment by anthonykulis** [comment]
I dont like it. You do not sort every time and you have code files all over the place. If I needed to import them, I have to spend time digging. Its best to create a directory, maybe name it `builders` and then put each builder under there. Something. The way it is now, @ClarkLindsay, your team will have 3 separate paths for each developer. Come up with a scheme that they will all follow that makes sense to the person using your builders

**Comment by ClarkLindsay** [comment]
Looks good

**Pull request merged**